pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "partial-span-processor"
+name = "otel-partial-python"
```

Comment:
I don't think we should change the name of the package - I may be wrong though!

Reply:
fixed